openstudiocore/src/utilities/sql/SqlFile_Impl.cpp

```diff
@@ -82,46 +83,50 @@ namespace openstudio{
-            "CREATE TABLE ReportMeterExtendedData (ReportMeterExtendedDataIndex INTEGER PRIMARY KEY, MaxValue REAL, MaxMonth INTEGER, MaxDay INTEGER, MaxHour INTEGER, MaxStartMinute INTEGER, MaxMinute INTEGER, MinValue REAL, MinMonth INTEGER, MinDay INTEGER, MinHour INTEGER, MinStartMinute INTEGER, MinMinute INTEGER);"
-            "CREATE TABLE ReportVariableData (TimeIndex INTEGER, ReportVariableDataDictionaryIndex INTEGER, VariableValue REAL, ReportVariableExtendedDataIndex INTEGER);"
-            "CREATE TABLE ReportVariableDataDictionary(ReportVariableDataDictionaryIndex INTEGER PRIMARY KEY, VariableType TEXT, IndexGroup TEXT, TimestepType TEXT, KeyValue TEXT, VariableName TEXT, ReportingFrequency TEXT, ScheduleName TEXT, VariableUnits TEXT);"
-            "CREATE TABLE ReportVariableExtendedData (ReportVariableExtendedDataIndex INTEGER PRIMARY KEY, MaxValue REAL, MaxMonth INTEGER, MaxDay INTEGER, MaxHour INTEGER, MaxStartMinute INTEGER, MaxMinute INTEGER, MinValue REAL, MinMonth INTEGER, MinDay INTEGER, MinHour INTEGER, MinStartMinute INTEGER, MinMinute INTEGER);"
-            "CREATE TABLE RoomAirModels (ZoneIndex INTEGER PRIMARY KEY, AirModelName TEXT, AirModelType INTEGER, TempCoupleScheme INTEGER, SimAirModel INTEGER);"
-            "CREATE TABLE Schedules (ScheduleIndex INTEGER PRIMARY KEY, ScheduleName TEXT, ScheduleType TEXT, ScheduleMinimum REAL, ScheduleMaximum REAL);"
             "CREATE TABLE Simulations (SimulationIndex INTEGER PRIMARY KEY, EnergyPlusVersion TEXT, TimeStamp TEXT, NumTimestepsPerHour INTEGER, Completed BOOL, CompletedSuccessfully BOOL);"
-            "CREATE TABLE StringTypes (StringTypeIndex INTEGER PRIMARY KEY, Value TEXT);"
-            "CREATE TABLE Strings (StringIndex INTEGER PRIMARY KEY, StringTypeIndex  INTEGER, Value TEXT);"
-            "CREATE TABLE Surfaces (SurfaceIndex INTEGER PRIMARY KEY, SurfaceName, ConstructionIndex INTEGER, ClassName TEXT, Area REAL, GrossArea REAL, Perimeter REAL, Azimuth REAL, Height REAL, Reveal REAL, Shape INTEGER, Sides INTEGER, Tilt REAL, Width REAL, HeatTransferSurf INTEGER, BaseSurfaceIndex INTEGER, ZoneIndex INTEGER, ExtBoundCond INTEGER,  ExtSolar INTEGER, ExtWind INTEGER);"
-            "CREATE TABLE SystemSizes (SystemName TEXT, Description TEXT, Value REAL, Units TEXT);"
-            "CREATE TABLE TabularData (ReportNameIndex INTEGER, ReportForStringIndex INTEGER, TableNameIndex INTEGER, SimulationIndex INTEGER, RowNameIndex INTEGER, ColumnNameIndex INTEGER, RowId INTEGER, ColumnId INTEGER, Value TEXT, UnitsIndex INTEGER);"
+            "CREATE TABLE EnvironmentPeriods ( EnvironmentPeriodIndex INTEGER PRIMARY KEY, SimulationIndex INTEGER, EnvironmentName TEXT, EnvironmentType INTEGER, FOREIGN KEY(SimulationIndex) REFERENCES Simulations(SimulationIndex) ON DELETE CASCADE ON UPDATE CASCADE );"

```

Comment:
@mbadams5 this does not require any change but maybe we should put the new index keys at the end of the column list?  this might affect people using index based queries?